doc/releases/migration-guide-4.0.rst

```diff
@@ -278,6 +278,16 @@ Bluetooth Audio
   :c:func:`bt_bap_unicast_server_unregister` has been called.
   (:github:`76632`)
 
+* The Broadcast Audio Scan Service (BASS) can now be registered and unregistered dynamically
```

Comment:
This should also mention that `bt_bap_scan_delegator_register_cb` has been removed/merged with `bt_bap_scan_delegator_register`

Reply:
Done.